docs/api-doc.md

```
@@ -0,0 +1,129 @@
+
+| Method | Path       | 설명           | 성공 status |
+| ------ | ---------- | -------------- | ----------- |
+| POST   | /cards     | 카드 등록      | 201         |
+| GET    | /cards     | 카드 목록 조회 | 200         |
+| DELETE | /cards/:id | 카드 삭제      | 204         |
+
+## POST /cards
+
+요청
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)